bucket/spotx.json

```
@@ -0,0 +1,13 @@
+{
+    "version": "1.2",
+    "description": "Blocks ads and updates for the desktop version of Spotify and more.",
+    "homepage": "https://github.com/amd64fox/SpotX/",
+    "license": "MIT",
+    "url": "https://github.com/amd64fox/SpotX/releases/download/1.2/Install.bat",
+    "hash": "fa1c227614596f6eb930c08566a7d46af9b2590be78562cb2059bb6230cf7d4e",
+    "post_install": "cd $env:USERPROFILE\\scoop\\apps\\spotx\\1.2\\; .\\Install.bat",
```

```suggestion
    "url": "https://raw.githubusercontent.com/amd64fox/SpotX/main/Install.ps1",
    "hash": "830e64d8e9c2d0cceea389ff2123d3ea19edc9ce1783e1b0de12c78fac4f128f",
    "installer": {
        "script": [
            "& \"$dir\\Install.ps1\" -confirm_uninstall_ms_spoti -confirm_spoti_recomended_over -podcasts_off -cache_off -block_update_on",
            "Remove-Item \"$([Environment]::GetFolderPath('Desktop'))\\Spotify.lnk\""
        ]
    },
    "uninstaller": {
        "script": "Invoke-ExternalCommand \"$Env:Appdata\\Spotify\\Spotify.exe\" -ArgumentList @('/uninstall', '/silent') | Out-Null"
    },
```

`Install.bat` cannot install the app **silently**. This will be a problem if the user is installing/updating multiple apps with *Scoop*.

[Install_Auto.bat](https://github.com/amd64fox/SpotX/blob/main/scripts/Install_Auto.bat) is not completely silent as well. (still needs to close *Spotify* and press Enter to continue)

The only way I can think of is to use `Install.ps1`, which is the main program itself.